addons/pos_restaurant_stripe/static/src/js/payment_stripe.js

```diff
@@ -0,0 +1,21 @@
+odoo.define('pos_restaurant_stripe.payment', function (require) {
+    "use strict";
+
+    var PaymentStripe = require('pos_stripe.payment');
+
+    PaymentStripe.include({
+        captureAfterPayment: async function (processPayment, line) {
```

Comment:
Why not use short methods syntax?

Reply:
I didn't make a conscious decision to use this syntax, I based it off of `pos_restaurant_adyen` and that's what it uses. Is there an advantage except less bytes? There's no consistency in POS modules at the moment ([payment.js](https://github.com/odoo/odoo/blob/277d71d56cecc71a98a418e1d3636faf23f3b748/addons/pos_restaurant/static/src/js/payment.js) even uses both in the same file). If somebody decides which one to use in POS modules perhaps a single commit can change all of them to be consistent.